backend/models/bookModel.js

```diff
@@ -0,0 +1,48 @@
+const mongoose = require('mongoose')
+
+const Schema = mongoose.Schema;
+
+const bookSchema = new Schema({
+    title: {
+        type: String,
+        required: true,
+        unique: true,
+        trim: true
+    },
+    author:{
+        type: String,
+        required: true,
+    },
+    description: {
+        type: String,
+        required: true,
+    }, 
+    bookImg:{
+        type: String,
+        required: true,
+    },
+    genre: {
+        type:String
+    },
+    tags:{
+        type: String,
+    },
+
+    fav: {
+        type: Boolean,
+        default: true,
```

Comment:
it's defaulted to be a favourite?

Reply:
should it be defaulted to false?

Reply:
in general, booleans are defaulted to false unless there's a reason. why would the user want every book to be auto-favourited?

Reply:
idk if they just added the quote, doesnt it mean they want to rmb?